### ISSUE TITLE
`npx init` fails with 401 Unauthorized

### ISSUE THREAD

**Issue by pieterbeulque** [wxt-dev/wxt#943]
### Describe the bug

Trying to init a new project fails on a `loadTemplates` step, for some reason.

```
Need to install the following packages:
wxt@0.19.7
Ok to proceed? (y)

npm warn deprecated inflight@1.0.6: This module is not supported, and leaks memory. Do not use it. Check out lru-cache if you want a good and tested way to coalesce async requests by a key value, which is much more comprehensive and powerful.
npm warn deprecated rimraf@2.4.5: Rimraf versions prior to v4 are no longer supported
npm warn deprecated glob@6.0.4: Glob versions prior to v9 are no longer supported

WXT 0.19.7                                                                                                                                           10:10:36 AM
ℹ Initalizing new project                                                                                                                           10:10:36 AM

 ERROR  Request failed with status 401 Unauthorized                                                                                                  10:10:37 AM

  at listTemplates (~/.npm/_npx/a24d67041fb1761b/node_modules/wxt/dist/core/initialize.mjs:79:13)
  at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  at async initialize (~/.npm/_npx/a24d67041fb1761b/node_modules/wxt/dist/core/initialize.mjs:9:21)
  at async cli.command.option.option.action.wrapAction.disableFinishedLog (~/.npm/_npx/a24d67041fb1761b/node_modules/wxt/dist/cli/commands.mjs:88:7)
  at async CAC.<anonymous> (~/.npm/_npx/a24d67041fb1761b/node_modules/wxt/dist/cli/cli-utils.mjs:16:22)
  at async ~/.npm/_npx/a24d67041fb1761b/node_modules/wxt/dist/cli/index.mjs:11:1

✖ Command failed after 463 ms                                                                                                                       10:10:37 AM

 ERROR  Failed to load templates                                                                                                                     10:10:37 AM

  at listTemplates (~/.npm/_npx/a24d67041fb1761b/node_modules/wxt/dist/core/initialize.mjs:90:11)
  at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
  at async initialize (~/.npm/_npx/a24d67041fb1761b/node_modules/wxt/dist/core/initialize.mjs:9:21)
  at async cli.command.option.option.action.wrapAction.disableFinishedLog (~/.npm/_npx/a24d67041fb1761b/node_modules/wxt/dist/cli/commands.mjs:88:7)
  at async CAC.<anonymous> (~/.npm/_npx/a24d67041fb1761b/node_modules/wxt/dist/cli/cli-utils.mjs:16:22)
  at async ~/.npm/_npx/a24d67041fb1761b/node_modules/wxt/dist/cli/index.mjs:11:1

### Reproduction

Run `npx wxt@latest init my-name` 

### Steps to reproduce

`npx wxt@latest init my-name`

### System Info

```shell
System:
    OS: macOS 14.6
    CPU: (12) arm64 Apple M3 Pro
    Memory: 97.30 MB / 18.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 20.16.0 - ~/.nvm/versions/node/v20.16.0/bin/node
    Yarn: 1.22.22 - ~/.nvm/versions/node/v20.16.0/bin/yarn
    npm: 10.8.2 - ~/.nvm/versions/node/v20.16.0/bin/npm
    pnpm: 8.15.8 - ~/Library/pnpm/pnpm
    bun: 1.1.25 - /opt/homebrew/bin/bun
    Watchman: 2024.08.19.00 - /opt/homebrew/bin/watchman
  Browsers:
    Chrome: 128.0.6613.86
    Safari: 17.6
```

### Used Package Manager

npm

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/wxt-dev/wxt/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://wxt.dev/guide/installation.html).
- [X] Check that there isn't [already an issue](https://github.com/wxt-dev/wxt/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/wxt-dev/wxt/discussions) or join our [Discord Chat Server](https://discord.gg/ZFsZqGery9).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by pieterbeulque** [comment]
Also breaks with `pnpm`

**Comment by aklinker1** [comment]
Released fix in [`v0.19.8`](https://github.com/wxt-dev/wxt/releases/tag/wxt-v0.19.8)
